### PR TITLE
[bitnami/kibana] Fix Kibana tests that started failing due to a major release

### PR DESCRIPTION
**Pull request by superaleks**
Signed-off-by: alukic <alukic@vmware.com>

### Description of the change
There was a major Kibana release which forces the CSP which is a problem for Cypress tests. We need to disable CSP so that the tests will run without issues.
Also, there are small changes in the UI which the tests need to adapt to. 

This is tested on the fork with the [following executions.](https://github.com/superaleks/charts/runs/6206267985?check_suite_focus=true)
### Checklist

- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by superaleks** [comment]
I'll do this from the fork.

File: .vib/kibana/cypress/cypress/integration/kibana_spec.js [hunk 1]
```diff
@@ -1,68 +1,58 @@
...
-  cy.get('[title*="Index pattern created is complete"]').should(
-    'be.visible',
-  );
+      `${td.testIndexName}.${random}`
+    );
+  });
+  cy.get('[data-test-subj="dataVisualizerFileImportButton"]').click();
+  cy.get('[title*="Index created"]').scrollIntoView().should('be.visible');
+  cy.get('[title*="Data uploaded"').scrollIntoView().should('be.visible');
+  cy.get('[title*="Index created"]').should('be.visible');
```
Comment (joancafom) [inline .vib/kibana/cypress/cypress/integration/kibana_spec.js:26]: This assertion is the same one as line `24`. Looking at the old code, maybe you meant `Index pattern created`  instead?

Comment (superaleks) [reply .vib/kibana/cypress/cypress/integration/kibana_spec.js]: Good catch, it should actually say `Data view created`.
I've modified the assertions and removed the extra `should('be.visible')` assertions. Please check now.

**Review by migruiz4 — APPROVED** [review]
LGTM!

**Review by joancafom — APPROVED** [review]

**Pull request merged**